src/storage/v2/disk/storage.cpp

```diff
@@ -825,6 +988,26 @@ StorageInfo DiskStorage::GetInfo() const {
   return {vertex_count, edge_count, average_degree, utils::GetMemoryUsage(), GetDiskSpaceUsage()};
 }
 
+void DiskStorage::SetEdgeImportMode(EdgeImportMode edge_import_status) {
+  std::unique_lock main_guard{main_lock_};
+  if (edge_import_status == edge_import_status_) {
+    return;
+  }
+  if (edge_import_status == EdgeImportMode::ACTIVE) {
+    edge_import_mode_cache_ = std::make_unique<EdgeImportModeCache>(config_);
+  } else {
+    edge_import_mode_cache_ = nullptr;
```

Comment:
How can we disable import mode?
Should we here reset unique_ptr and only after that set it to nullptr?

Reply:
reset(nullptr) should be the way to go I think

Reply:
Yes, you're right: https://en.cppreference.com/w/cpp/memory/unique_ptr/operator%3D